### ISSUE TITLE
Scannet validation set

### ISSUE THREAD

**Issue by OPradelle** [huaaaliu/RGBX_Semantic_Segmentation#17]
Hi, thanks for sharing your work.

I tried to use the ScanNet pretrained model to obtain the validation set mIoU score but the network gives poor result.
I generated the HHA image from the raw depth and use the images at a resolution of 640x480.
Did i miss something there ?

Thanks for your answer

**Comment by OPradelle** [comment]
Hi, thanks for your answer.

I used directly the code you provide and update only on the config file (ie. data path, image size) using the configuration provided in your paper for the ScanNet dataset. Can you provide the config file used for this task to check on my side if anything went wrong ?

Looking at the HHA images, it look like the same provided in your paper, but i want to know if you first inpaint the depth to fill the empty value or if you directly use the raw depth.

Thanks.
